content/essentials/signals.mdx

```diff
@@ -0,0 +1,105 @@
+In this context, if the value of the signal changes, it notifies its dependencies of this change. 
+As a result, anything that is subscribed to this signal by accessing it within a tracking scope, will be re-evaluated or re-run. 
+This ensures that anything dependent on this signal will always be in sync with the state represented by these signals.
+
+<Aside>
+
+A tracking scope is typically created by [`createEffect`](/reference/createEffect)  or [`createMemo`](/reference/createMemo), other Solid primitives. 
+Both functions subscribe to the signals accessed within them, establishing a dependency relationship.
+
+When a signal within this scope changes its value, these dependencies are identified, and the functions or computations relying on them are triggered to update.
+
+</Aside>
+
+## Updating values
+
+To update a signal's value, you just need to invoke the setter function returned by `createSignal`:
+
+```jsx
+setCount(count()+1)
+
+console.log(count()) // output: 1
+```
+
+When the setter is called, the value of the signal is automatically updated, triggering the required updates in the functions that are tracking that signal.
+
+## Reacting to changes
+
+A distinct feature of signals is their ability to keep a list of their dependents. 
+[Effects](/essentials/effects) and [memos](/essentials/memos) subscribe to signals, inherently becoming _dependent _on them.
+This ensures a streamlined updating process across all the dependent functions whenever the signal's value changes.
+
+```jsx
+function DependentCounter() {
+  const [count, setCount] = createSignal(0);
+  
+  // logs initial value of count
+  console.log(count()) // result: 0
+  
+  // creates a reactive value `double` that is always twice the value of `count`
+  const double = createMemo(() => count() * 2);
+  
+  return (
+    <div>
+      <p>Count: {count()}</p>
+      <p>Double: {double()}</p>
+      <button onClick={() => setCount(count() + 1)}>Increment</button>
+    </div>
+  );
+}
+
+export default DependentCounter;
+```
+In this example, `createSignal` initializes a reactive state variable `count` and sets its initial value to `0` .
+
+In the intialization code surrounding the `createSignal`, including the `console.log` statement, is executed only _once_, when the component is initially rendered.
+This means that the initial value of `count`, which is `0`, will be logged to the console only once, regardless of subsequent changes to `count`.
+
+On the other hand, `double` is initialized using `createMemo`, another Solid primitive, that creates a reactive computed value based on `count` signal.
+Unlike the `console.log` statement, the `createMemo` function ensures that `double`  tracks the changes in `count`. 
+This means that whenever `count`  is updated, `double`  automatically recalculates and the component will re-render to display the updated value.
```

Comment:
Maybe worth mentioning how it would have worked without the explicit `createMemo()` wrapper, but while still keeping `count() * 2` wrapped in a function. Especially since in practice that memo basically serves no point, so the reader might think that it's necessary, maybe.